docs/3.api/1.composables/use-cookie.md

```diff
@@ -11,7 +11,7 @@ const cookie = useCookie(name, options)
 ```
 
 ::alert{icon=👉}
-`useCookie` only works during `setup` or `Lifecycle Hooks`.
+`useCookie` only works during `setup`, plugins, and `Lifecycle Hooks`.
```

Comment:
```suggestion
`useCookie` only works during `setup`, `plugins`, and `Lifecycle Hooks`.
```

should we remove \` on "`Lifecycle Hooks`" or add it to "plugins" ?

Reply:
the other issue here with 'lifecycle hooks' is that really there are some (vue) caveats here:

using these kind of composables after an await or outside certain lifecycle hooks can also cause memory leaks. (e.g. `watch` can't be auto-cleaned up).

Reply:
Maybe we can add a warning to life cycle hooks that it's generally a less optimal approach and that it's better to use in plugins and setup?